packages/govuk-frontend/src/govuk/components/checkboxes/checkboxes.mjs

```diff
@@ -40,21 +40,32 @@ export class Checkboxes extends GOVUKFrontendComponent {
     /** @satisfies {NodeListOf<HTMLInputElement>} */
     const $inputs = $module.querySelectorAll('input[type="checkbox"]')
     if (!$inputs.length) {
-      return this
+      throw new ElementError(null, {
+        componentName: 'Checkboxes',
+        identifier: '<input type="checkbox">'
+      })
     }
 
     this.$module = $module
     this.$inputs = $inputs
 
     this.$inputs.forEach(($input) => {
       const targetId = $input.getAttribute('data-aria-controls')
+      const targetElement = document.getElementById(targetId)
 
-      // Skip checkboxes without data-aria-controls attributes, or where the
-      // target element does not exist.
-      if (!targetId || !document.getElementById(targetId)) {
+      // Skip radios without data-aria-controls attributes
+      if (!targetId) {
         return
       }
 
+      // Throw if target conditional element does not exist.
+      if (!targetElement) {
+        throw new ElementError(targetElement, {
+          componentName: 'Checkboxes',
+          identifier: `.govuk-checkboxes__conditional #${targetId}`
```

Comment:
I like how this is clearer, but realised we only select by ID?

```suggestion
          identifier: `#${targetId}`
```

Reply:
I do prefer CSS selectors though, see https://github.com/alphagov/govuk-frontend/pull/4236/files#r1331775855

Reply:
I'm happy to revert this for this PR, but something to discuss when we review the content - is `targetId` user-friendly enough when there may be some kind of issue with it? Ideally having the parent radios item info in there would be great, but realise I may be going too far.

Reply:
Did some digging, and we used to scope it to `$module` before https://github.com/alphagov/govuk-frontend/commit/266f8772857ca9afc60ded16279ea4b2c9595462

I suppose since then the target could genuinely be anywhere on the page?

E.g. Tick a box in a left column and a question is revealed on the right?